src/libDirectoryService/ViewChangePreProcessing.cpp

```diff
@@ -454,13 +454,13 @@ bool DirectoryService::NodeVCPrecheck() {
       return true;
     }
 
-    for (const auto& dsblock : m_vcPreCheckDSBlocks) {
+    /*for (const auto& dsblock : m_vcPreCheckDSBlocks) {
```

Comment:
Why still preserve the unnecessary code?

Reply:
actually I want JH to review this

Reply:
@bb111189 Could you please review on this to see if this piece of code necessary to keep?

Reply:
Why remove this piece of code?